06-process.Rmd

```diff
@@ -320,6 +320,7 @@ The following chart, taken from [@ncbi] shows re-admission rates for different p
 * **Other important KPIs**: Some of the other important KPIs for healthcare providers include, patient satisfaction scores, asset utilization for operating theatres, diagnostic and other equipment and patient footfalls.
 
 **In the publich health sector, a few of the common data being generated are**:
```

Comment:
line 322: Spelling of public
line 344:  **analyzing** the prevalence of **diseases**

Reply:
Hey @pragyansharma24, the changes you mentioned are out of the scope of issue #1354 
I have worked on the parts that were highlighted in the issue and not the overall section.

Reply:
@gauravhassija Spelling should not be out of scope. Please fix and merge.

Reply:
Fixed the spelling errors